apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/Chat.tsx

```diff
@@ -305,7 +306,10 @@ export function TokenUsage({
       )}
     >
       {responseStream === undefined ? (
-        <Text.H6M color='foregroundMuted'>{tokens} tokens</Text.H6M>
+        <Text.H6M color='foregroundMuted'>
+          Prompt tokens: {usage?.promptTokens} | Completion tokens:{' '}
+          {usage?.completionTokens}
+        </Text.H6M>
```

Comment:
How looks this? Can you put a screenshot?

Reply:
<img width="296" alt="image" src="https://github.com/user-attachments/assets/5cd55300-319e-4df7-af6a-0d24ecdc13a0">
<img width="299" alt="image" src="https://github.com/user-attachments/assets/0029670e-0520-457c-9e8b-8fe9a3a910d4">

Reply:
Hey @Ashad-h thank you so much for your contribution! I really like the detailed count but I'm afraid it might be too much info for the user.

What do you think about keeping the general count as it was and then showing this detailed count in a tooltip when hovering the original element?

Reply:
Tip: here's an example of using our [Tooltip component](https://github.com/latitude-dev/latitude-llm/blob/65a58f110ac0e13bd0a91027d659470a0b515de6/apps/web/src/app/(private)/datasets/generate/GenerateDatasetContent.tsx#L302)

Reply:
Yes good idea ! What about something like this:

![Untitled](https://github.com/user-attachments/assets/14d9a94e-6967-4b9f-be3c-de002b130597)

Reply:
Yes, that's nice. The only thing put side='top'

Reply:
Done :)

Reply:
Lovely!